css/styles.css

```
@@ -108,7 +108,8 @@ h6 {
 }
 
 #github-chart-img {
-	width: 70rem;
+	/* width: 70rem; */
```

```suggestion
	/* width: 70rem; */
```

Remove commented past values